webapp/markdown.py

```diff
@@ -1,7 +1,38 @@
-from mistune import Markdown, BlockLexer
+from mistune import BlockGrammar, BlockLexer, Markdown, _pure_pattern
+import re
+
+
+class DescriptionGrammar(BlockGrammar):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.list_block = re.compile(
+            r"^( *)(?=[*+-•]|\d+\.)(([*+-•])?(?:\d+\.)?) [\s\S]+?"
```

Comment:
Can this be unit tested in some way?
We are adding a very specific custom functionality to markdown parser. This is a HUGE and complicated regexp that I'm not even trying to fully understand. It would be great to have some test cases showing what this thing is doing.

As far as I understand we are taking text input (description) and turning it into HTML. Can we test that particular text input (with different types of bullets) ends up in HTML we expect?

Reply:
test added

Reply:
Thanks!